Makefile

```diff
@@ -650,8 +650,8 @@ release-notes:
 #########
 
 .PHONY: debug-deploy
-debug-deploy: codegen-install ## Install debug manifests
-	@kubectl create -f ./config/install_debug.yaml || kubectl replace -f ./config/install_debug.yaml
+debug-deploy: codegen-manifest-debug ## Install debug manifests
+	@kubectl create -f ./.manifest/debug.yaml || kubectl replace -f ./.manifest/debug.yaml
```

Comment:
Nit: shall we switch to server side apply ?

Reply:
Can you please elaborate further on what you mean?

Reply:
`kubectl apply -f ./.manifest/debug.yaml  --server-side`

Reply:
It will not add the offending annotation in this case.